examples/reminderbot/domain.yml

```diff
@@ -13,7 +13,7 @@ intents:
 - EXTERNAL_reminder
 - EXT_reminder
 entities:
-- name
+- PERSON
```

Comment:
In keeping with recommendations in the rest of the doc, I changed reminderbot to use spacy for names